README.md

```diff
@@ -3,5 +3,9 @@ ASP.NET Docs
 
 This project provides the source for [docs.asp.net](http://docs.asp.net/). You can learn more about ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
 
-**We accept pull requests!** However, before submitting a pull request, please [read the CONTRIBUTING guidelines](CONTRIBUTING.md), which include information on how to build the docs locally, as well as style and organizational guidance.
+~~**We accept pull requests!** However, before submitting a pull request, please [read the CONTRIBUTING guidelines](CONTRIBUTING.md), which include information on how to build the docs locally, as well as style and organizational guidance.~~
```

Comment:
I wouldn't leave this crossed out section. It looks like we changed our mind on accepting community contributions, which we haven't.